src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const DAILY_FEE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let sum = days * DAILY_FEE;
+
+  if (days >= LONG_TERM) {
+    return (sum -= LONG_TERM_DISCOUNT);
```

Comment:
The parentheses around `sum -= LONG_TERM_DISCOUNT` are unnecessary. You can simply return `sum - LONG_TERM_DISCOUNT` since `sum` is not used after this point.